cogs/slowmode.py

```diff
@@ -8,9 +8,29 @@
 
 import util
 
+import time
+
 HELPER_ROLE = os.getenv('HELPER_ROLE')
 MOD_ROLE = os.getenv('MOD_ROLE')
 
+CHANNELS = [190327462087884811, 766335071590023247]
+
+message_cache = {}
+previous_delays = {}
+last_updated = 0
+
+
+UPDATE_FREQUENCY = 30
+time_configs = {
+    # messages per UPDATE_FREQUENCY : slowmode delay
+    5: 0,
+    10: 1,
+    20: 3,
+    30: 5,
+    40: 10,
+    50: 15,
```

Comment:
1. this might need to be a bit more aggressive: 50 messages in 30 seconds is already very much to handle, and 15 second slowmode is not much more than the 10 second default we usually have.
2. maybe this could be put within the function of get_delay?
3. [global variables should be all caps camel case](https://peps.python.org/pep-0008/#constants)

Reply:
alternatively, would be nice if this were configurable during runtime

Reply:
1. could you provide reasonable times?
2. put dict into function
3. made variable all caps
4. will add a command to configure it